crates/node/src/service/block_production.rs

```diff
@@ -79,7 +79,10 @@ impl Service for BlockProductionService {
                     .context("Building genesis block from devnet config")?;
 
                 block_import
-                    .add_block(genesis_block, BlockValidationContext::new(backend.chain_config().chain_id.clone()))
+                    .add_block(
+                        genesis_block,
+                        BlockValidationContext::new(backend.chain_config().chain_id.clone()).trust_class_hashes(true),
```

Comment:
if you add that here, can you also add it in devnet/src/lib.rs in the tests?

Reply:
yes I'll also remove any from_random usage on the tests to match what we have in the rest of the codebase

Reply:
i dont think there are any :)
we use the DevnetKeys struct directly to access the secrets